database/mssql/scripts/versions/v_7_ddl.sql

```diff
@@ -0,0 +1,66 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+CREATE TABLE [permit].[ORBC_TRANSACTION](
+	[TRANSACTION_ID] [bigint] IDENTITY(1,1) NOT NULL,
+	[MERCHANT_ID] [int] NOT NULL,
```

Comment:
We should leave the MERCHANT_ID out of the database, and only have it as a configuration parameter in the vault. I have a suspicion it's *mildly* sensitive and we should be able to retrieve the merchant id from bambora with a lookup later on using the bambora transaction id. This is something it would be good to get Doug F.'s input on as well.

Reply:
I agree. The MERCHANT_ID should not be stored in our database.